save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Profile.kt

```diff
@@ -142,36 +168,32 @@ val profileSettingsCard: FC<SettingsProps> = FC { props ->
 }
 
 /**
- * @param props
- * @param avatarImgLink
- * @param setAvatarImgLink
- * @param placeholder
  * @param avatarWindowOpen
  */
 internal fun ChildrenBuilder.avatarEditor(
-    props: SettingsProps,
-    avatarImgLink: String?,
     avatarWindowOpen: WindowOpenness,
-    setAvatarImgLink: StateSetter<String?>,
-    placeholder: String,
 ) {
-    label {
-        className = ClassName("btn animated-provider")
+    Link {
+        className = ClassName("btn px-0 pt-0")
         title = AVATAR_TITLE
         onClick = {
             avatarWindowOpen.openWindow()
         }
```

Comment:
Are you sure you want to use `Link` with `onClick` handler? Better use button for this purpose.